### ISSUE TITLE
[hr_attendance] Cannot scan barcode with phone/laptop camera

### ISSUE THREAD

**Issue by aleuffre** [odoo/odoo#118799]
## Impacted versions: 
16

## Steps to reproduce: 

Install hr_attendance.
In the settings, set up the "Attendance Mode" as "Barcode/RFID". Also pick front or back camera for the barcode source.
Open Kiosk mode.

## Current behavior:

Nothing happens. Odoo doesn't even ask for camera permissions, and it certainly doesn't read anything from either camera.

## Expected behavior:

I can scan barcodes using my camera on my phone or laptop.

## Additional notes

I see the options were added here. https://github.com/odoo/odoo/pull/99622

I suspect these are just hooks for an Enterprise module and they do not do anything in Community, except be misleading as to what Odoo can actually do, but maybe I just don't understand how they're supposed to work?

**Comment by altela** [comment]
Hello,
I would also like to let you know that installing hr_attendance in either on-premise or docker makes it unable to trigger the camera (the camera button is missing)

This is what expected when kiosk mode of attendance in run (captured from runbot)

![Screen Shot 2023-08-22 at 17 23 16](https://github.com/odoo/odoo/assets/68892527/c5d4571b-2d24-4e21-8964-32636df50cd4)

Odoo.sh is able to show the button just like the above. Somehow, on-premise or docker Odoo was unable to show the camera thus user cannot trigger it

<img width="494" alt="Screen Shot 2023-08-22 at 17 25 59" src="https://github.com/odoo/odoo/assets/68892527/f2db057d-1bcf-492b-8504-21dd821986c5">

**Comment by bnisevic** [comment]
I have this issue even with SSL.

**Comment by Mahaboob929** [comment]
yeah i tried with odoo17 by installing ssl using certbot it worked but not in odoo16 what is the reason

**Comment by bnisevic** [comment]
@Mahaboob929 I think that the main reason is **underdeveloped** feature for the community version. Because this feature works perfectly on version 18.

**Comment by Mahaboob929** [comment]
can anyone tell me how to solve the issue
